ui/imports/shared/views/chat/LinksMessageView.qml

```diff
@@ -263,6 +264,31 @@ ColumnLayout {
         }
     }
 
+    Component {
+        id: unfurledProfileLinkComponent
+        UserProfileCard {
+            id: unfurledProfileLink
+            readonly property var contact: Utils.parseContactUrl(parent.link)
+            readonly property var contactDetails: Utils.getContactDetailsAsJson(contact.publicKey)
+
+            onContactChanged: console.log("!!!!!Contact changed",JSON.stringify(contact))
+            onContactDetailsChanged: console.log("!!!!!Contact details changed",JSON.stringify(contactDetails))
```

Comment:
Debug leftover?

Reply:
Ohh, yes

Reply:
Oh and also, when you give that contact a local nickname, shouldn't the card react and auto-reload itself? That could give some meaningful code to the handlers above :)

Reply:
> Oh and also, when you give that contact a local nickname, shouldn't the card react and auto-reload itself? That could give some meaningful code to the handlers above :)

I'm not sure yet, but I think that it shouldn't. This just a temporary implementation to test the user profile links. AFAIK in the final implementation the user profile will be fetched by the sender and we will show only what's in a waku message (the waku message data could easily get outdated when the user profile data changes). @igor-sirotin please correct if I'm wrong.

Reply:
> Debug leftover?

Fixed